src/main/cli/commandLine.ts

```diff
@@ -0,0 +1,92 @@
+// ==LICENSE-BEGIN==
+// Copyright 2017 European Digital Reading Lab. All rights reserved.
+// Licensed to the Readium Foundation under one or more contributor license agreements.
+// Use of this source code is governed by a BSD-style license
+// that can be found in the LICENSE file exposed on Github (readium) in the project repository.
+// ==LICENSE-END==
+
+import { readerActions } from "readium-desktop/common/redux/actions";
+import { OpdsFeedRepository } from "readium-desktop/main/db/repository/opds";
+import { PublicationRepository } from "readium-desktop/main/db/repository/publication";
+import { container } from "readium-desktop/main/di";
+import { RootState } from "readium-desktop/main/redux/states";
+import { CatalogService } from "readium-desktop/main/services/catalog";
+import { Store } from "redux";
+import { URL } from "url";
+import { isArray } from "util";
+
+export async function cli_(filePath: string[] | string) {
+
+    let publicationOpenRequested = false;
+    let returnValue = true;
+    const filePathArray = isArray(filePath) ? filePath : [filePath];
+
+    for (const fp of filePathArray) {
+        // import and read publication
+        const catalogService = container.get("catalog-service") as CatalogService;
+        const publication = await catalogService.importFile(fp);
+        const store = container.get("store") as Store<RootState>;
+        if (publication) {
+            if (!publicationOpenRequested) {
```

Comment:
@llemeurfr @mutterback
Is there a use case for the Thorium CLI to bulk-import the given filepath parameters? (for instance, eKitabu catalog provisioning?)
For example: `thorium.exe PATH1 PATH2 PATH3 ... PATHXX` or `thorium.exe FOLDER`

If so, this would seem like the right place to take into account a `silent` boolean flag, which would skip the step "open the first successfully-imported publication into a reader view" (note that with the current PR code, subsequent successfully-imported publications are not opened)

Also, the combination of `paths` parameter + the `silent` flag would negate the need for the special `import` command (which is the equivalent of `silent` === `true`).

Reply:
In no case "thorium.exe PATH1 PATH2 PATH3 ... PATHXX" is the import command

I will just add an option to add multiple publication in a same time and/or a glob path.

It will be `./Thorium import PATH1 PATH2 ...``
or/and `./Thorium 'myPath/**/*.epub' to import all the folder


The command ./Thorium PATH1 is mainly for "open with" on windows/linux/mac

Reply:
I'm ok for adding a bulk import, as e.g. `./Thorium import 'myPath/**/*.epub'`.
It will allow for instance the import of every IPDF samples in one command. 

I don't see a use case for importing the content of multiple folders.

Reply:
So right now, if a user invokes the command line `thorium.exe PATH1 PATH2 PATH3 ... PATHXX` (assuming all paths exist) =>
1) all paths are imported
2) PATH1 is open in the reader view

So why have a separate `import` CLI function? This seems redundant. Why not provide a `silent` boolean flag (default `false`) so that users can bulk-import a list of publications without triggering the "open reader for first successful import"?

Reply:
PS: +1 to Glob support.
https://github.com/sindresorhus/globby
or
https://github.com/isaacs/node-glob

Reply:
The Idea behind défault command is to have a fallback. Normally on défault mode : it have to just read and not import to library .. but we can't doing this yet

Reply:
Yes this would be great: temporarily open/read a publication without importing into the library ... but I think this would require a pretty significant refactoring for LCP support, reading location/bookmarks/annotations etc. (anything that requires the reader state to be managed via the DB)

Reply:
... or simply delete the publication when the app is closed, maybe.

Reply:
Yes, I think this is the most pragmatic / realistic option. This would probably cause UX confusion though (e.g. publication in bookshelf whilst app is open, then disappears at next restart) ... unless we have a special flag which hides the "transient" EPUBs from the library.

Reply:
You can remove publication after reading, but that implies to track the closing or 'goto library' events from reader window.

The issue is more deeper, how implement both duplicate or replace the publication ? we need the publication identifier. When the user import several ebooks they actually duplicated.

Reply:
Yes, see Laurent's suggestions here: https://github.com/readium/readium-desktop/issues/438